docs/BUILD.md

```
@@ -137,3 +137,56 @@ make -j4
 # To use it as a command, uncomment the following line:
 # sudo make install -j4
 ```
+
+# How to build with MinGW/MSYS2
+
+Install MSYS2: https://www.msys2.org/
+
+#### Clone this repo recursively:
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)